File: tests/fixtures/structural-index/kotlin/src/BadgeConsumer.kt
package fixture

import fixture.renderBadge

fun badgeSummary(label: String): String {
    // renderBadge mentioned in a comment
    val ignored = "renderBadge in a string"
    return renderBadge(label)
}

fun themeAccent(): String = BadgeTheme.accent

fun cardSummary(renderer: BadgeRenderer, label: String): String {
    return BadgeCard(renderer).summarize(label)
}

File: tests/fixtures/structural-index/swift/src/BadgeConsumer.swift
import func BadgeKit.renderBadge

func badgeSummary(_ label: String) -> String {
    // renderBadge mentioned in a comment
    let ignored = "renderBadge in a string"
    return renderBadge(label)
}

func themeAccent() -> String {
    return BadgeTheme.accent
}

func cardSummary(_ card: BadgeCard) -> String {
    return card.summarize("unit")
}
